salt/metalk8s/salt/master/files/salt-master-pod.yaml.j2

```diff
@@ -0,0 +1,86 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: salt-master
+  namespace: kube-system
+  labels:
+    app: salt-master
+    app.kubernetes.io/name: salt-master
+    app.kubernetes.io/version: '{{ salt_master_version }}'
+    app.kubernetes.io/component: salt
+    heritage: metalk8s
+    app.kubernertes.io/part-of: metalk8s
+    app.kubernetes.io/managed-by: salt
+spec:
+  hostNetwork: true
+  priorityClassName: system-cluster-critical
+  securityContext:
+    runAsUser: 0
+    fsGroup: 0
+    readOnlyRootFilesystem: true
+    allowPrivilegeEscalation: false
+    capabilities:
+      drop:
+        - ALL
+  containers:
+    - name: salt-master
+      image: {{ registry_prefix }}{{salt_master_image}}:{{salt_master_version}}
+      ports:
+        - name: publisher
+          containerPort: 4505
+          protocol: TCP
+        - name: requestserver
+          containerPort: 4506
+          protocol: TCP
+      livenessProbe:
+        exec:
+          command:
+          - pgrep
+          - salt-master
+        periodSeconds: 10
+        initialDelaySeconds: 5
+      readinessProbe:
+        exec:
+          command:
+          - salt-run
```

Comment:
considering the master ready to accept if its MQ is accepting events

Reply:
it is more agressive too we dont want to push events into the bus every 10 seconds